src/main/java/mate/academy/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -24,9 +24,7 @@ public class MovieSessionDaoImpl implements MovieSessionDao {
     @Override
     public MovieSession add(MovieSession movieSession) {
         Transaction transaction = null;
-        Session session = null;
-        try {
-            session = HibernateUtil.getSessionFactory().openSession();
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
```

Comment:
why did you do this?

Reply:
I didn't change the classes that I wasn't supposed to by the task.
Just forked and implemented stuff according to the README.md.